--- xpra-2.5.3/setup.py~	2022-04-04 08:41:27.000000000 +0200
+++ xpra-2.5.3/setup.py	2022-04-04 08:43:53.267150608 +0200
@@ -30,7 +30,7 @@
 
 if sys.version<'2.7':
     raise Exception("xpra no longer supports Python 2 versions older than 2.7")
-if sys.version[0]=='3' and sys.version<'3.4':
+if sys.version_info[:3] < (3, 4, 0):
     raise Exception("xpra no longer supports Python 3 versions older than 3.4")
 #we don't support versions of Python without the new ssl code:
 if not hasattr(ssl, "SSLContext"):
