--- xpra-0.7.8/./xpra/webm/__init__.py~	2013-01-01 09:26:41.000000000 +0200
+++ xpra-0.7.8/./xpra/webm/__init__.py	2013-01-28 15:42:14.857716211 +0200
@@ -41,7 +41,7 @@
     _LIBRARY_NAMES = ["libwebp_a.dll"]
 
 elif sys.platform == "linux2":
-    _LIBRARY_NAMES = ["libwebp.so", "libwebp.so.2"]
+    _LIBRARY_NAMES = ["libwebp.so", "libwebp.so.2", "libwebp.so.4"]
 
 elif sys.platform == "darwin":
     _LIBRARY_NAMES = ["libwebp.dylib"]
