--- xpra-2.5.3/setup.py.orig	2021-06-30 11:17:57.958945510 +0200
+++ xpra-2.5.3/setup.py	2021-06-30 16:12:21.894265170 +0200
@@ -1539,10 +1539,7 @@
         if scripts_ENABLED:
             scripts += ["scripts/xpra_udev_product_version", "scripts/xpra_signal_listener"]
         libexec_scripts = []
-        if is_Fedora() or is_CentOS() or is_RedHat():
-            libexec = "libexec"
-        else:
-            libexec = "lib"
+        libexec = "@libexec@"
         if xdg_open_ENABLED:
             libexec_scripts += ["scripts/xdg-open", "scripts/gnome-open", "scripts/gvfs-open"]
         if server_ENABLED:
