--- xpra-5.0.4/xpra/codecs/evdi/evdi_compat.h.orig	2023-11-28 15:59:56.000000000 +0100
+++ xpra-5.0.4/xpra/codecs/evdi/evdi_compat.h	2024-02-01 19:27:23.737745673 +0100
@@ -12,7 +12,9 @@ void evdi_connect_compat(evdi_handle han
           const unsigned int edid_length,
           const uint32_t pixel_area_limit,
           const uint32_t pixel_per_second_limit) {
-#if LIBEVDI_VERSION_MAJOR>1 || LIBEVDI_VERSION_MINOR>11
+#if LIBEVDI_VERSION_MAJOR>1 || LIBEVDI_VERSION_MINOR>=14
+	return evdi_connect2(handle, edid, edid_length, pixel_area_limit, pixel_per_second_limit);
+#elif LIBEVDI_VERSION_MAJOR>1 || LIBEVDI_VERSION_MINOR>11
 	return evdi_connect(handle, edid, edid_length, pixel_area_limit, pixel_per_second_limit);
 #else
 	return evdi_connect(handle, edid, edid_length, pixel_per_second_limit);
